addons/website/views/website_templates.xml

```diff
@@ -34,9 +34,10 @@
 <template id="assets_editor" name="Editor assets">
     <script type="text/javascript" src="/website/static/src/js/website.editor.js"></script>
     <script type="text/javascript" src="/website/static/lib/jQuery.transfo.js"></script>
-    <script type="text/javascript" src="/website/static/lib/ace/ace.js"></script>
-    <script type="text/javascript" src="/website/static/lib/ace/theme-monokai.js"></script>
-    <script type="text/javascript" src="/website/static/lib/ace/mode-xml.js"></script>
+    <script type="text/javascript" src="/web/static/lib/ace/ace.js"></script>
+    <script type="text/javascript" src="/web/static/lib/ace/theme-monokai.js"></script>
+    <script type="text/javascript" src="/web/static/lib/ace/mode-xml.js"></script>
+    <script type="text/javascript" src="/web/static/lib/ace/mode-python.js"></script>
```

Comment:
Well, the issue was that, ace.js having dependency over mode-python, so I have to load this js file, but what I am suggesting instead of having dependecy on mode-python, mode-xml, and theme-monokai  why not loading all this modes separately.
That is remove dependency from ace_call, here we kept dependency just because ace_call also calls load of ace_mode_xml, ace_theme_monokai and ace_mode_python.
So we should require, ace_load, ace_mode_xml, ace_theme_monokai, ace_mode_python in form_widget.js and website_ace.js and call load of each module manually.
What do you think ?